admin_manual/configuration_server/logging_configuration.rst

```diff
@@ -43,7 +43,7 @@ been configured by the **datadirectory** parameter in :file:`config/config.php`.
 
 The desired date format can optionally be defined using the **logdateformat** parameter in :file:`config/config.php`.
 By default the `PHP date function`_ parameter ``c`` is used, and therefore the
-date/time is written in the format ``2013-01-10T15:20:25+02:00``. By using the
+date/time is written in the format ``2013-01-10T15:20:25+02:00``. This format complies to RFC3339 without time zone and with numeric time zone offset. The respective GO time format is ``2006-01-02T15:04:05+0700``. By using the
```

Comment:
Not sure if mandatory, but for consistency I suggest to add line breaks.
```suggestion
date/time is written in the format ``2013-01-10T15:20:25+02:00``. This format
complies to RFC3339 without time zone and with numeric time zone offset.
The respective GO time format is ``2023-01-10T15:20:25+0200``. By using the
```
I also aligned GO format time. Looks confusing when the "respective" GO format example shows a different time than the other examples.

Reply:
The GO date format relies on certain numbers to be used as representatives for e.g. the month within the given string.
-> https://gosamples.dev/date-time-format-cheatsheet/

The GO time format does not represent a certain example date but encodes the structure of where which data is given.

P.S.: i missed a ":" at "+02:00"

Reply:
So it is exactly the same format as the PHP default. Then I do not understand the purpose of this PR/sentence at all.

Reply:
2006-01-02T15:04:05+07:00 would be the correct GO date format where numbers used in the date have a specific meaning.
It is therefore different to what is stated in the documentation.